src/main/scala/LumberJack.scala

```diff
@@ -16,17 +16,35 @@ object LumberJack extends App
     case firstAryElem =>  firstAryElem foreach{secondAryElem => print(secondAryElem.toString + " ")}
       print('\n')}
 
-  println("")
   var logAmount: Int = 7
+  println("")
+
+  var samllestNum = findsmallestNum(currentPiles)
+
+
+    for (i <- 0 to pileAmount.size- 1)
+    {
+      logAmount = adjustLogPile(pileAmount , logAmount , findsmallestNum(currentPiles), i)
+      println(logAmount)
+    }
+
 
-  while(logAmount > 0 )
-  {
-    pileAmount = adjustLogPile(pileAmount, genrateDisplay(currentPiles, pileAmount) , logAmount)
-  }
   currentPiles = genrateDisplay(currentPiles, pileAmount)
+
   currentPiles foreach{case a =>  a foreach{b => print(b.toString + " ")}; print('\n') }
 
+  def adjustLogPile(pileAmount: Array[Int], logAmount: Int , smallestNum: Int , i: Int): Int = {
+    var j:Int = logAmount
```

Comment:
please name this something like "numberOfLogs" or something that's not one character